examples/interpreter-demo/async-execution.html

```diff
@@ -43,166 +42,140 @@ <h1>Asynchronous Execution with JS-Interpreter</h1>
-                    }
-                  },
-                  "inputs": {
-                    "VALUE": {
-                      "block": {
-                        "type": "math_arithmetic",
-                        "fields": {
-                          "OP": "MULTIPLY"
-                        },
-                        "inputs": {
-                          "A": {
-                            "block": {
-                              "type": "variables_get",
-                              "fields": {
-                                "VAR": {
-                                  "id": ",4tsL~{roFQW!-CL1(Gn"
-                                }
-                              }
-                            }
-                          },
-                          "B": {
-                            "block": {
-                              "type": "math_number",
-                              "fields": {
-                                "NUM": 2
-                              }
-                            }
-                          }
-                        }
-                      }
-                    }
-                  },
-                  "next": {
-                    "block": {
-                      "type": "text_print",
-                      "id": "print",
-                      "inputs": {
-                        "TEXT": {
-                          "block": {
-                            "type": "variables_get",
-                            "fields": {
-                              "VAR": {
-                                "id": ",4tsL~{roFQW!-CL1(Gn"
-                              }
-                            }
-                          }
-                        }
-                      }
-                    }
-                  }
-                }
-              }
-            }
-          }
-        }
-      },
-      {
-        "type": "variables_set",
-        "x": 0,
-        "y": 0,
-        "fields": {
-          "VAR": {
-            "id": ",4tsL~{roFQW!-CL1(Gn"
-          }
-        },
-        "inputs": {
-          "VALUE": {
-            "block": {
-              "type": "math_number",
-              "fields": {
-                "NUM": 1
-              }
-            }
-          }
-        },
-        "next": {
-          "block": {
-            "type": "controls_repeat_ext",
-            "inputs": {
-              "TIMES": {
-                "block": {
-                  "type": "math_number",
-                  "fields": {
-                    "NUM": 4
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-    ]
-  },
-  "variables": [
-    {
-      "name": "n",
-      "id": ",4tsL~{roFQW!-CL1(Gn"
-    }
-  ]
-}
     var demoWorkspace = Blockly.inject('blocklyDiv',
         {media: 'https://unpkg.com/blockly/media/',
-         toolbox: toolboxJson});
+         toolbox: document.getElementById('toolbox')});
```

Comment:
Why did you switch it back to using XML rather than JSON? Was this an accidental reversion?

Reply:
The XML toolbox contains the 'wait' block this demo needs.  The JSON toolbox (used by the other demos) does not have this block.  Not having the 'wait' block renders this demo useless.

Reply:
Can you add the 'wait' block to the JSON toolbox?

Reply:
What's the advantage of using JSON?  The JSON toolbox looks significantly larger.  And our developer tools only generates XML.

Reply:
We are trying to standardize to using JSON toolboxes, because (1) the XML serialization system is iceboxed, (2) the XML toolbox does not work well within certain frameworks (e.g. React), and (3) consistency with all of our other demos.